xbmc/video/VideoDatabase.cpp

```diff
@@ -3625,6 +3667,151 @@ void CVideoDatabase::SetMovieSet(int idMovie, int idSet)
     ExecuteQuery(PrepareSQL("update movie set idSet = null where idMovie = %i", idMovie));
 }
 
+void CVideoDatabase::GetMovieVersion(int idMovie, CFileItemList& items)
```

Comment:
Could this method be `const`?

Reply:
The ConstructPath is non const, so can't directly make this const. Just made both ConstructPath and SplitPath const to fix this.